prSHARK/backends/github.py

```diff
@@ -17,12 +18,18 @@ class Github:
     However, since every pull request is an issue in github we also fetch events and comments from the issue endpoint.
     """
 
-    def __init__(self, config, project, pull_request_system):
+    def __init__(self, config, project):
         self.config = config
         self._log = logging.getLogger('prSHARK.github')
 
-        self._prs = pull_request_system
         self._p = project
+        self.last_system_id = None
+        self.pr_system = None
+        self.parsed_prs = {'prs': {}, 'reviews': {}, 'review_comments': {}, 'comments': {}, 'files': {}, 'events': {}}
```

Comment:
Just a remark. When we discussed this I was thinking more along the lines of a Python dataclass to hold the structures.
That way you could have used type hints to help IDEs.